src/rgw/rgw_group.h

```diff
@@ -0,0 +1,132 @@
+#ifndef CEPH_RGW_GROUP_H
+#define CEPH_RGW_GROUP_H
```

Comment:
if a source file includes rgw_group.h and nothing else, it should be able to compile. some dependencies are missing that would prevent that from working:

- forward declarations for `CephContext` and `RGWRados`
- `string` and `vector` should be qualified with `std::`, with the `<string>` and `<vector>` headers included
- needs `include/encoding.h` (i think?) for `WRITE_CLASS_ENCODER`